donut/modules/bodfeedback/helpers.py

```diff
@@ -0,0 +1,248 @@
+from donut import email_utils
+from donut.modules.bodfeedback import email_templates
+import flask
+import pymysql.cursors
+
+
+def send_update_email(email, complaint_id):
+    # Sends an email to [email] of poster
+    msg = email_templates.added_message.format(get_link(complaint_id))
+    subject = "Received BoD Feedback"
+    email_utils.send_email(email, msg, subject)
+    # Notifies BoD of new complaint
+    email_utils.send_email("bod@donut.caltech.edu", msg, subject)
+
+
+def register_complaint(data, notification=True):
+    '''
+    Inputs a complaint into the database and returns the complaint id
+    associated with this complaint
+    data should be a dict with keys 'course', 'msg' and optionally 'name', 'email'
+    if required fields are missing, returns false
+    '''
+    if not (data and data['subject'] and data['msg']): return False
+    # Register complaint
+    query = """
+    INSERT INTO bod_complaint_info (subject, status, uuid)
+    VALUES (%s, %s, UUID())
+    """
+    status = 'new_msg'
+    with flask.g.pymysql_db.cursor() as cursor:
+        cursor.execute(query, (data['subject'], status))
+    complaint_id = cursor.lastrowid
+    # Add email to db if applicable
+    if data['email']:
+        emails = [x.strip() for x in data['email'].split(',')]
+        for e in emails:
+            add_email(complaint_id, e, False)
```

Comment:
This iterates over the list of emails twice. Would be better to do something like:
```python
for email in data['emails'].split(','):
    add_email(complaint_id, email.strip(), False)
```